### PR TITLE
Updated README with required OS versions

### PR DESCRIPTION
**Pull request by ilijapuaca**

### CODE REVIEW

**Comment by ilijapuaca** [comment]
Added an example gif to README, is that too big?

**Comment by ibgreen** [comment]
> Added an example gif to README, is that too big?

No I don't think it is stored in the repo.

File: README.md [hunk 1]
```diff
@@ -26,6 +28,21 @@ Many features normally considered fundamental by deck.gl applications are not ev
...
+- iOS13+ is required when building for a simulator, as `Metal` is available on the simulator starting with iOS13
+- Currently has to be built with bitcode disabled (under investigation)
+
+### macOS
+
+- macOS10.14+
+
+#### Linux
+
+- Tested and runs on Ubuntu Bionic
```
Comment (isaacbrodsky) [inline README.md:45]: Well, not sure we have graphics working on Ubuntu. I think we need to do something to switch Ubuntu from Vulkan to OpenGL to get it to render.

Comment (ilijapuaca) [reply README.md:45]: It shouldn't be using Vulkan to begin with according to build settings, so I'm kind of confused about that one.

We could likely figure this one out pretty quickly, either if you have some spare time to take a look, or I could set up a Linux box once we start wrapping things up

**Comment by ilijapuaca** [comment]
> No I don't think it is stored in the repo.

I meant the actual gif size in the README, whether it takes up too much space up at the top :)

**Pull request merged**